spec/models/proposal_spec.rb

```diff
@@ -314,13 +314,24 @@
         end
       end
 
+      context "with a blank reason" do
+        let(:reason) { ' ' }
```

Comment:
what about just assigning `reason` within the `it` block!? (I see that would be inconsistent with the rest of this file...so not necessary, but certainly an organizational pattern I think we should be moving toward!!